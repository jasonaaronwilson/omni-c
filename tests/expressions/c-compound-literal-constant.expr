(value_result_t){0, .u64=0};
