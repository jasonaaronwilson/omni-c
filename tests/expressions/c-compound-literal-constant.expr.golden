{
    tag = PARSE_NODE_COMPOUND_LITERAL
    type_node = {
        tag = PARSE_NODE_TYPE
        type_node_kind = TYPE_NODE_KIND_TYPENAME
        type_name = value_result_t
        type_args = [
        ]
    }
    initializers = [
        {
            tag = PARSE_NODE_LITERAL
            tokens = [
                0
            ]
        }
        {
            tag = PARSE_NODE_DESIGNATED_INITIALIZER
            member_name = u64
            value = {
                tag = PARSE_NODE_LITERAL
                tokens = [
                    0
                ]
            }
        }
    ]
}

// C Output
(value_result_t){0, .u64=0}
